src/main/resources/application.yaml

```diff
@@ -570,6 +570,15 @@ govnotify:
     requestNewHearingRequirements:
       legalRep:
         email: fd709bf4-095c-4734-8f88-9a9feba9fd81
+    recordRemissionDecision:
+      partiallyApproved:
+        legalRep:
+          email: 717b0290-36eb-4adf-a4c5-baacf59d379d
+        adminOfficer:
+          email: 954c2959-e13a-4907-be1e-3eb9831fc68c
+      rejected:
+        legalRep:
+          email: 8a48b422-c6ed-448d-aec0-a5b43bd2b472
```

Comment:
Similar to above, you need to add placeholders to this template for customer services telephone number and email address.

Reply:
Updated template in Gov notify